### ISSUE TITLE
Evaluation Metrics Higher than Reported: Possible Issue?

### ISSUE THREAD

**Issue by xiao10ma** [chengweialan/DeSiRe-GS#8]
Thank you very much for your excellent work!

I followed the instructions in your `README.md` to train `084` for both stage 1 and stage 2, and used your provided script for evaluation. However, the metrics I obtained are better than those reported in the paper. Could there be something I missed or did incorrectly?

Here are my results:

```sh
+--------+-------+-------+-------+----------------+----------------+
|  PSNR  | SSIM  | LPIPS |  L1   | PSNR (dynamic) | SSIM (dynamic) |
+========+=======+=======+=======+================+================+
| 37.272 | 0.965 | 0.027 | 0.009 | 37.430         | 0.965          |
+--------+-------+-------+-------+----------------+----------------+
```

**Comment by manggiechen** [comment]
> Thank you very much for your excellent work!非常感谢您的出色工作！
> 
> I followed the instructions in your `README.md` to train `084` for both stage 1 and stage 2, and used your provided script for evaluation. However, the metrics I obtained are better than those reported in the paper. Could there be something I missed or did incorrectly?我按照您 `README.md` 中的说明为第 1 阶段和第 2 阶段训练了 `084`，并使用您提供的脚本进行评估。但是，我获得的指标比论文中报告的要好。会不会有什么我遗漏或做错了什么？
> 
> Here are my results:  以下是我的结果：
> 
> +--------+-------+-------+-------+----------------+----------------+
> |  PSNR  | SSIM  | LPIPS |  L1   | PSNR (dynamic) | SSIM (dynamic) |
> +========+=======+=======+=======+================+================+
> | 37.272 | 0.965 | 0.027 | 0.009 | 37.430         | 0.965          |
> +--------+-------+-------+-------+----------------+----------------+

Hello, I have the same problem. Do you have any conclusions?

**Comment by xiao10ma** [comment]
作者分辨率比 PVG 低一半，[issue](https://github.com/chengweialan/DeSiRe-GS/issues/13) @sunsonji @chengweialan

**Comment by FudongGe** [comment]
> 作者分辨率比 PVG 低一半，[issue](https://github.com/chengweialan/DeSiRe-GS/issues/13) [@sunsonji](https://github.com/sunsonji) [@chengweialan](https://github.com/chengweialan)

Hi, do you know which resolution corresponds to the results in Table 1 of the paper? What I understand is that PVG is 1920 * 1280, but desire-gs is 960 * 640? Or, the results in Table 1 do not correspond to the current code for reducing resolution？

**Comment by manggiechen** [comment]
> > 作者分辨率比 PVG 低一半，[issue](https://github.com/chengweialan/DeSiRe-GS/issues/13) [@sunsonji](https://github.com/sunsonji) [@chengweialan](https://github.com/chengweialan)
> 
> Hi, do you know which resolution corresponds to the results in Table 1 of the paper? What I understand is that PVG is 1920 * 1280, but desire-gs is 960 * 640? Or, the results in Table 1 do not correspond to the current code for reducing resolution？

Indeed, the direct comparison of different rates of difference is also strange to me.You'll find that some of the contrasting models are 960*640 resolution .and it is not clear whether the authors have done some kind of average.

**Comment by FudongGe** [comment]
> > > 作者分辨率比 PVG 低一半，[issue](https://github.com/chengweialan/DeSiRe-GS/issues/13) [@sunsonji](https://github.com/sunsonji) [@chengweialan](https://github.com/chengweialan)
> > 
> > 
> > Hi, do you know which resolution corresponds to the results in Table 1 of the paper? What I understand is that PVG is 1920 * 1280, but desire-gs is 960 * 640? Or, the results in Table 1 do not correspond to the current code for reducing resolution？
> 
> Indeed, the direct comparison of different rates of difference is also strange to me.You'll find that some of the contrasting models are 960*640 resolution .and it is not clear whether the authors have done some kind of average.

Yes, I'm also a bit confused. Comparing under the same resolution condition is meaningful.
